actors/evm/src/interpreter/system.rs

```diff
@@ -91,6 +91,11 @@ pub struct System<'r, RT: Runtime> {
     bytecode: Option<EvmBytecode>,
     /// The contract's EVM storage slots.
     slots: StateKamt<RT::Blockstore>,
+
+    /// The contract's EVM transient storage slots.
+    transient_slots: StateKamt<RT::Blockstore>,
+    pub(crate) transient_data_lifespan: Option<TransientDataLifespan>,
```

Comment:
Does this need `pub(crate)`?

Reply:
(I'd actually like to fix the visibility of the other cases as well, but they're a bit trickier.

Reply:
a98c0585 remove pub visibility from transient_data_lifespan. TODO still is combine transient_slots with transient_data_lifespan anyway